posts/benchmarking_design_decisions/index.qmd

```diff
@@ -0,0 +1,266 @@
+---
+title: "Using benchmarking to guide the adoption of dependencies in R packages"
+author:
+  - name: "James Mba Azam"
+    orcid: "https://orcid.org/0000-0001-5782-7330"
+date: last-modified
```

Comment:
The `date` field will need to be updated to a static date before publishing to avoid re-publishing this post if we ever fix a typo or an URL in it.

Reply:
I've fixed that here acf2ede453b5ea607d98db1feaeb21863530e6ce.